npm/npm_import.bzl

```diff
@@ -89,118 +89,23 @@ def npm_translate_lock(
         # buildifier: disable=unused-variable
         warn_on_unqualified_tarball_url = None,
         **kwargs):
-    """Repository rule to generate npm_import rules from pnpm lock file or from a package.json and yarn/npm lock file.
+    """Repository macro to generate `npm_import` rules from a lock file.
```

Comment:
Quite a bit of generated code in `@npm` repository besides just the `npm_import` rules. The generated `package_json.bzl` pass-throughs and the aliases in the root BUILD file for example.

Reply:
"Repository macro to generate starlark code from a lock file"